.env

```diff
@@ -152,9 +151,7 @@ DATE_FORMAT="Y-m-d"
 
 ###> base url ###
 # Base URL for client callbacks
-BASE_URL=""
-# Webpack public path
-WEBPACK_PUBLIC_PATH="/build"
```

Comment:
Does Helm require that you set these valuse in .env rather than .env.local? Removing this variable will default the build path to `/build` (as seen in https://github.com/ucfopen/UDOIT/blob/dev/v3-3-1/webpack.config.js#L13). If `BASE_URL` is set to `http://127.0.0.1:8000/udoit3`, then `WEBPACK_PUBLIC_PATH` needs to be set to `/udoit3/build`

Or do these changes even matter, since you're configuring everything through the `values.yml`?

Reply:
I must admit I do not recall why this was removed/changed at all. Looking. at the commits, it seems the `WEBPACK_PUBLIC_PATH` variable was not in the original branch we worked off of? https://github.com/ucfopen/UDOIT/pull/848/commits/9519c9ec4b512197c70402b84fe9842e9fd6d482

I may be misunderstanding.

In any case for our purposes, they do not actually matter. I will fix this to match what is there now